.github/workflows/ci.yml

```diff
@@ -144,12 +144,12 @@ jobs:
           sudo xcodebuild -downloadPlatform visionOS
           sudo xcodebuild -runFirstLaunch
       - name: Lint Podspec
-        run: bundle exec pod lib lint --verbose --fail-fast --swift-version=6.0 --platforms=${{ matrix.platforms }}
```

Comment:
this version comes from the `.swiftformat` file – duplicating here is a recipe for a future failed build

Reply:
oops. this is cocoa pods! not the linter!